examples/shader-coverage-backends/vk_compute_demo.cpp

```
@@ -0,0 +1,426 @@
+{
+    if (descPool)
+        vkDestroyDescriptorPool(device, descPool, nullptr);
+    if (cmdPool)
+        vkDestroyCommandPool(device, cmdPool, nullptr);
+    if (device)
+        vkDestroyDevice(device, nullptr);
+    if (instance)
+        vkDestroyInstance(instance, nullptr);
+}
+
+Buffer Context::createBuffer(VkDeviceSize size, VkBufferUsageFlags usage)
```

Cppcheck (2.21.0)

[style] The function 'createBuffer' is never used.

(unusedFunction)